### PR TITLE
Function to extract texture features #1350

### PR DESCRIPTION
**Pull request by charleygros**
### Requirements

* [x] Is there a issue open describing the motivation for this pull request ?
https://github.com/neuropoly/spinalcordtoolbox/issues/1350

### CODE REVIEW

File: batch_processing_hawaii17.sh [hunk 1]
```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
```
Comment (peristeri) [inline batch_processing_hawaii17.sh]: This file is not needed in this pull request

**Comment by charleygros** [comment]
@jcohenadad : In the very new pushed version of test_sct_analyze_texture: we compute only one texture feature, instead of 6 in the past --> I think that it speed up the testing

Let me know if it is sufficient, otherwise I will compute the texture only on few slices as you suggested.

**Comment by jcohenadad** [comment]
@charleygros ok, it runs in 10s now, which is acceptable. We can go with this, if you are confident that not testing all other texture features might not be a problem in the future.

**Comment by charleygros** [comment]
"if you are confident that not testing all other texture features might not be a problem in the future" --> Yes I am pretty confident of that: it is just a loop over the scikit-image functions (one round instead of 6 round)

**Comment by jcohenadad** [comment]
ok i guess we can merge once it passes Travis. Good job!!

**Pull request merged**